packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1592,6 +1621,14 @@
 	"contextual.workflows.sharing.unavailable.button.cloud": "Upgrade now",
 	"contextual.workflows.sharing.unavailable.button.desktop": "View plans",
 
+	"contextual.variables.unavailable.title": "Available on Enterprise plan",
+	"contextual.variables.unavailable.title.cloud": "Available on Power plan",
+	"contextual.variables.unavailable.title.desktop": "Upgrade to n8n Cloud to collaborate",
```

Comment:
ideally we should have separate files, but the same keys and load those variants instead... `en-cloud.json` and `en-desktop.json`. tech debt that could be addressed elsewhere